deploy/kubernetes/tracee.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  labels:
+    app.kubernetes.io/name: tracee
+    app.kubernetes.io/instance: tracee-v1
```

Comment:
this instance label is meaningless if we put the same value in all instances, so let's remove it